Feat  rdf active feedback (#92)
ImperialCollegeLondon/icl-rcs-user-guide: docs/rdfactive/rdfactive-faq.md

///

/// details | How much do I need to know about ReCAP to use the RDF-Active storage?
    type: faq
Most users of the RDF-Active only need a cursory knowledge of ReCAP in order to actually use the facility. The only key piece of information needed is the storage allocation's file path, which can be obtained even by word of mouth instead of directly accessing ReCAP itself. 

Managers of research groups, however, should read and review our [documentation for ReCAP](../recap/index.md) as understanding and being fluent in [access control](../recap/research-groups.md#user-management) is a critical skill and part of their leadership responsibility to carry out.
///

/// details | Why isn't the RDF-Active accessible from HPC systems?
    type: faq
As part of a wider restructuring of RCS services in order to improve system stability, targetted and intentional services, which should result in higher user satisfaction.

> Over five years of running the RDS service has taught us that while centralising all research data on a single platform is convenient, supporting the full spectrum of workloads on one system is extremely challenging and often degrades service for everyone. We notice this was exasperated by the challenges around HPC workloads in particular. So we've decided to move to dedicated, purpose‑built storage platforms, called the Research Data Facilities (RDF), where we customise systems for specific use cases. This shift also removes physical location constraints suffered by the RDS, giving us more flexibility in where these systems are deployed and scaled.

Reviewer: ```suggestion
Over five years of running the RDS service has taught us that while centralising all research data on a single platform is convenient, supporting the full spectrum of workloads on one system is extremely challenging and often degrades service for everyone. We noticed this was exasperated by the challenges around HPC workloads in particular. So we've decided to move to dedicated, purpose‑built storage platforms, called the Research Data Facilities (RDF), where we customise systems for specific use cases. This shift also removes physical location constraints suffered by the RDS, giving us more flexibility in where these systems are deployed and scaled.
```

> To migrate your data out from the RDS, Globus will be the primary method used whether the destination facility is the RDF-Active, HX2, or any other of our newer and more robust facilities. (See [here](./rds-rdfactive-migration.md) for further details on migration.)

Reviewer: ```suggestion
To migrate your data out from the RDS, Globus will be the primary method used whether the destination facility is the RDF-Active, HX2, or any other of our newer and more robust facilities. (See [here](./rds-rdfactive-migration.md) for further details on migration.)
```
So it is less to do with migration and more to do with moving data around generally. The RDS->RDF-Active migration when we do for users is done using a different tool we only have short term access to just for the purpose

Author: I'm not sure what to do with this information, or if specifying what you've mentioned is of any benefit to the users. Maybe removing this sentence entirely is the move? Whilst keeping the bracketed part.

> For research groups that require shared spaces on HPC platforms — for collaborative workflows, permissions‑managed access to common datasets, and consistent data governance — we will provide shared data storage areas upon request. To make a request, please [contact us](https://www.imperial.ac.uk/admin-services/ict/self-service/research-support/rcs/get-support/contact-us/) via our ticket system.

Reviewer: ```suggestion
For research groups that require shared spaces on HPC platforms — for collaborative workflows, permissions‑managed access to common datasets, and consistent data governance — we will provide shared data storage areas upon request. To make a request, please [contact us](../support/index.md) via our ticket system.
```
